### ISSUE TITLE
How to use kfac to train two probabilistic models jointly? 

### ISSUE THREAD

**Issue by wangleiphy** [deepmind/kfac-jax#53]
In my application, I need to jointly optimize two probabilistic models. They contribute to two different terms in the final loss function. 

I am wondering what would be the recommended pattern of using kfac ?  
More specifically, does it make sense to invoke  `kfac_jax.register_normal_predictive_distribution` twice (for the two probabilistic models respectively) ?

Thanks in advance!

**Comment by wangleiphy** [comment]
Perhaps a more concrete example of my question: 

How train a [VAE](https://github.com/deepmind/dm-haiku/blob/main/examples/vae.py) using `kfac-jax`?

**Comment by wangleiphy** [comment]
Thanks!
